tests/RDF/NodeUri/CRC32HashGeneratorTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\RDF\NodeUri;
+
+use PHPUnit\Framework\TestCase;
+
+final class CRC32HashGeneratorTest extends TestCase
+{
+    private CRC32HashGenerator $hashGenerator;
+
+    protected function setUp(): void
+    {
+        $this->hashGenerator = new CRC32HashGenerator();
+    }
+
+    /**
+     * @test
+     */
+    public function it_should_generate_produces_consistent_hash(): void
+    {
+        $data = [
```

Comment:
I would take a more realistic example, for example a translated address, or a complex booking information.

Furthermore, this is already a sorted array and it doesn't test the recursive implementation.